### PR TITLE
Import color theme from r2

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: libdec/printer.js [hunk 1]
```diff
@@ -16,6 +16,37 @@
...
+            "callname": "gray",
+            "integers": "cyan",
+            "comment": "red",
+            "labels": "green",
+            "types": "green",
+            "macro": "yellow",
+            "flow": "magenta",
+            "text": "yellow"
+        };
+        if (useR2Colors) {
```
Comment (wargio) [inline libdec/printer.js]: Change this into `if (r2cmd) {`

File: libdec/printer.js [hunk 2]
```diff
@@ -16,6 +16,37 @@
  */
 
 (function() {
+    function initializeColors() {
+        const useR2Colors = true; // hardcoded. TODO. add a config var
```
Comment (wargio) [inline libdec/printer.js]: remove this (see below)

File: libdec/colors/ansi.js [hunk 3]
```diff
@@ -27,16 +27,28 @@
...
-        }
         var fn = function(x) {
             var o = arguments.callee;
             return o.open + x + o.close;
         };
-        fn.open = '\u001b[' + __colors[name][0] + 'm';
-        fn.close = '\u001b[' + __colors[name][1] + 'm';
+        if (name.startsWith('rgb:')) {
+            name = name.substring (4);
+            const str = '\u001b[48;2;'+ pair(name, 0) + ';' + pair(name, 2) + ';' + pair(name, 4) + ';m'
```
Comment (wargio) [inline libdec/colors/ansi.js]: missing `;`

File: libdec/printer.js (hunk 1 above)
Comment (radare) [reply libdec/printer.js]: what about having a config option to pick that? shouldnt r2cmd be always available?

Comment (wargio) [reply libdec/printer.js]: not on the testsuite

File: libdec/printer.js [hunk 4]
```diff
@@ -16,6 +16,25 @@
...
+        });
+        return {
+            "callname": config['call'] || 'gray',
+            "integers": config['num'] || 'cyan',
+            "comment": config['comment'] || 'red',
+            "labels": config['flag'] || ' green',
+            "types": config['func_var_type'] || 'green',
+            "macro": config['cjmp'] || 'yellow',
+            "flow": config['flow'] || 'magenta',
+            "text": config['usrcmt'] || 'yellow,
```
Comment (wargio) [inline libdec/printer.js]: missing `'`

**Pull request merged**